### DISCUSSION TITLE
Multiple profiles

### DISCUSSION THREAD

**Discussion by RGBok** [Taewan-P/gpt_mobile#67] [Ideas]
Would be nice to be able to make multiple profiles per provider. For example, multiple profiles for open AI with different model and settings. With or without having same API key.

**Comment by Taewan-P** [comment]
This idea will be linked with https://github.com/Taewan-P/gpt_mobile/discussions/62
As in the comments, this needs major refactoring. So it might take some time.

**Comment by Taewan-P** [comment]
Hello, this feature is now implemented in [v0.7.0](https://github.com/Taewan-P/gpt_mobile/releases). Thanks!
